### DISCUSSION TITLE
Should we keep RefreshRateService working in order to use panel overdrive in g-helper?

### DISCUSSION THREAD

**Discussion by Zephyrus-7** [seerge/g-helper#722] [Q&A]
I wonder since Smart Display Control is not needed, is RefreshRateService obsolete or not with g-helper? And what's overdrive exactly, can anyone explain?

**Comment by seerge** [comment]
@CyberMan38 hello, your question is already addressed in readme https://github.com/seerge/g-helper#recommendations-optional

It's not needed.

Overdrive makes panel response faster https://www.displayninja.com/what-is-overdrive-on-a-monitor/
